ops/roles/ota/tasks/database.yml

```diff
@@ -13,7 +13,7 @@
     - update
     - setup
 
-- name: Obtain {{ name }} initial data from {{ repository }}
+- name: Obtain {{ name }} initial data from branch {{ branch }} of {{ repository }}
```

Comment:
Where is defined `branch` variable? I do not find it, I think it does not exist.

Reply:
Sorry, this should have been imported from https://github.com/ambanum/OpenTermsArchive/commit/b2cf6280691545af8e5c060f0c7a83773dc84935#diff-dedd8f47cf84f35c9e2aa4e11ba9e512921ac00f50ce1ca6deaefb7355fa964e!